### PR TITLE
Revert move v6 changes

### PR DESCRIPTION
**Pull request by oxade**
We're reverting this change because it can break encodings and we're too close to devnet 0.14 to guarantee test coverage

### CODE REVIEW

File: crates/sui/src/sui_move/build.rs [hunk 1]
```diff
@@ -41,7 +40,7 @@ impl Build {
             },
         )?;
         if dump_bytecode_as_base64 {
-            println!("{}", json!(pkg.get_package_base64()))
+            println!("{:?}", pkg.get_package_base64())
```
Comment (666lcz) [inline crates/sui/src/sui_move/build.rs:43]: This is a fix for the change in https://github.com/MystenLabs/sui/pull/5506. We should not revert this unless you also want to revert https://github.com/MystenLabs/sui/pull/5506

Comment (oxade) [reply crates/sui/src/sui_move/build.rs:43]: is this a one-liner? If so I can just change it.

**Review by 666lcz — APPROVED** [review]
The TS change looks good to me!

File: crates/sui/src/sui_move/build.rs (hunk 1 above)
Comment (666lcz) [reply crates/sui/src/sui_move/build.rs:43]: yes

Comment (oxade) [reply crates/sui/src/sui_move/build.rs]: Okay I put that back in.

**Pull request merged**